oidc-controller/api/core/oidc/issue_token_service.py

```diff
@@ -75,12 +77,13 @@ def get_claims(
             sub_id_value = sub_id_claim.value
 
         # add sub and append presentation_claims
-        claims.append(Claim(type="sub", value=sub_id_value))
-        claims += list(presentation_claims.values())
+        oidc_claims.append(Claim(type="sub", value=sub_id_value))
+        presentation_claims.values()
 
-        result = {c.type: c for c in claims}
-        logger.info("claims generated for the token:")
-        logger.info(result)
+        result = {c.type: c.value for c in oidc_claims}
+        result["vc_proof_claims"] = json.dumps(
```

Comment:
What about `vc_presented_attributes` as container name? This would be pretty self-explanatory and consistent with the Aries nomenclature as well. Thoughts?